SMLHelper/Handlers/LootDistributionHandler.cs

```diff
@@ -0,0 +1,133 @@
+﻿namespace SMLHelper.V2.Handlers
+{
+    using Interfaces;
+    using Patchers;
+    using SMLHelper.V2.Assets;
+    using System.Collections.Generic;
+    using System.Linq;
+    using UWE;
+
+    /// <summary>
+    /// A handler that manages Loot Distribution.
+    /// </summary>
+    public class LootDistributionHandler : ILootDistributionHandler
+    {
+        /// <summary>
+        /// Main entry point for all calls to this handler.
+        /// </summary>
+        public static ILootDistributionHandler Main { get; } = new LootDistributionHandler();
+
+        private LootDistributionHandler() { } // Hides constructor
+
+        /// <summary>
+        /// Adds in a custom entry into the Loot Distribution of the game.
+        /// You must also add the <see cref="WorldEntityInfo"/> into the <see cref="WorldEntityDatabase"/> using <see cref="WorldEntityDatabaseHandler"/>.
+        /// </summary>
+        /// <param name="data">The <see cref="LootDistributionData.SrcData"/> that contains data related to the spawning of a prefab, also contains the path to the prefab.</param>
+        /// <param name="classId">The classId of the prefab.</param>
+        public static void AddCustomLootDistData(LootDistributionData.SrcData data, string classId)
+        {
+            Main.AddCustomLootDistData(data, classId);
+        }
+
+        /// <summary>
+        /// Adds in a custom entry into the Loot Distribution of the game.
+        /// You must also add the <see cref="WorldEntityInfo"/> into the <see cref="WorldEntityDatabase"/> using <see cref="WorldEntityDatabaseHandler"/>.
```

Comment:
Would it be possible to combine all of these methods somehow into something that puts all the necessary data into one place?  
Maybe add new SMLHelper class that has properties for all of these items, just to make it easier to to everything necessary to add new things to the loot distribution.

Reply:
Yes, that would be `Fragment` class I was talking to you earlier about. Or are you talking about something else?